app/assets/stylesheets/openproject/_legacy.sass

```diff
@@ -593,7 +593,8 @@ h4.comment
     // Neccessary so that button and select box have the same height
     line-height: 0.6rem
 
-@include breakpoint(680px down)
+// Breakpoint for big mobile screens in landscape mode
+@include breakpoint(830px down)
```

Comment:
Is this really necessary? Same issue as above. Maybe we should discuss this in person later..